src/settings/SettingsModal.tsx

```diff
@@ -59,6 +60,8 @@ interface Props {
 export const SettingsModal = (props: Props) => {
   const { t } = useTranslation();
 
+  const { isEmbedded } = useUrlParams();
```

Comment:
I remember to also have seen some `!widget` logit for detecting embedded mode? Is this the proper way. Is this also consistend with all other code?

Reply:
For posterity: we decided to go with `isEmbedded` as it makes more sense for UI